etcd.conf.yml.sample

```diff
@@ -125,6 +125,9 @@ peer-transport-security:
   # Peer TLS using generated certificates.
   auto-tls: false
 
+# The validity period of the self-signed certificate, the unit is year.
```

Comment:
@tangcong glad you didn't forget this :) thanks!

Reply:
thanks. It can guide users how to set a reasonable expiration time, such as this [issue](#13208). We'd better also update the [document](https://github.com/etcd-io/website/blob/main/content/en/docs/v3.5/op-guide/configuration.md).

Reply:
Yup, agree. For etcd.io, I will create an issue/work on clean sweep with all new flags. Thanks!